packages/graphql-dynamodb-transformer/src/definitions.ts

```diff
@@ -315,6 +326,53 @@ export function makeModelXFilterInputObject(obj: ObjectTypeDefinitionNode): Inpu
     }
 }
 
+export function makeEnumFilterInputObject(
```

Comment:
This returns a list of object type definition nodes so lets make the functio name plural. `makeEnumFilterInputObjects`

Reply:
e2e tests already cover this use case. [DyamoDBMoldeTransformer](https://github.com/aws-amplify/amplify-cli/blob/master/packages/graphql-transformers-e2e-tests/src/__tests__/DynamoDBModelTransformer.e2e.test.ts#L39) has an `enum` type and that does get added to the filter.

I will follow the same approach taken for unit test